pages.ru/common/rm.md

```diff
@@ -0,0 +1,24 @@
+# rm
+
+> Удалить файлы или каталоги.
+> Больше информации: <https://www.gnu.org/software/coreutils/rm>.
+
+- Удалить файлы из определённых мест:
+
+`rm {{путь/до/файла}} {{путь/до/другого/файла}}`
```

Comment:
```suggestion
- Удалить файлы из определённых мест:

`rm {{путь/до/файла}} {{путь/до/другого_файла}}`
```

Mandatory fix (error source: original page): [broken path](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths): placeholder content conforming `path/to/[^/]*` regex expected.